### ISSUE TITLE
Update and publish

### ISSUE THREAD

**Issue by Dzarda7** [DustinWatts/FT6236#8]
Hello, first of all, thank you for this library. I forked it and found that it is also compatible with FT5436, so I just added chip IDs and made it compatible with both. I also added rotation functionality. It would be awesome if it will be in arduino library manager and PlatformIO libraries. If I open pull request, will you do it please? Else I can do it with my fork.

 Thank you so much.

**Comment by DustinWatts** [comment]
Hi @Dzarda7 Sure.. I will look at any pull request. And if there are no issues I will merge!

Thanks for your work on this!

**Comment by Dzarda7** [comment]
And will you publish it into library manager of arduino and PlatformIO? Thanks.

**Comment by marcelstoer** [comment]
> And will you publish it into library manager of Arduino and PlatformIO?

Just checked out this project and asked myself the same question 😄 Glad I don't need to create an issue (and PR as you already raised https://github.com/DustinWatts/FT6236/pull/9).

**Comment by DustinWatts** [comment]
@marcelstoer Lol... it always helps checking other issues 😉 

@Dzarda7 I don't know if I can push it in to the PlatformIO and Arduino manager, because there are already a a few and I used soms Adafruit code. I did credit them in the library but I don't know how this works publishing. I will figure this out.

**Comment by Dzarda7** [comment]
Thank you Dustin. It should be ok to publish it in my opinion, because all of it is under MIT licence. I can definitely understand that time management problem 😃. Please do that when you find some time and let me know I would really appreciate it.

**Comment by marcelstoer** [comment]
> there are already a a few

That's right, but...

- [LovyanGFX](https://github.com/lovyan03/LovyanGFX) is a bit of a monster to use IMO
- [htcw_ft6336](https://github.com/codewitch-honey-crisis/htcw_ft6336) and [FocalTech_Library](https://github.com/lewisxhe/FocalTech_Library) were created around the same time as your project and I assume you must have had a good reason not to use either of them

**Comment by DustinWatts** [comment]
@marcelstoer Yes. The reason is that this library is part of a much bigger project [FreeTouchDeck](https://www.github.com/dustinwatts/freetouchdeck) and this way I could fix and have control over most libraries used in that project.

**Comment by marcelstoer** [comment]
@DustinWatts #9 adds `library.json` required for the PlatformIO registry.

How to publish the library:
- for Arduino
  - https://github.com/arduino/library-registry/blob/main/FAQ.md#submission
  - it will then also automatically become listed at https://www.arduinolibraries.info/
  - once registered, you simply need to bump the version in `library.properties` and create a Git tag whenever you release a new version
- for PlatformIO
  - https://docs.platformio.org/en/latest/librarymanager/creating.html accompanied by 
  - https://youtu.be/W7FIJYhcAOk

Once #9 is merged the project is pretty much ready for submission. It won't take long.

**Comment by benlye** [comment]
Just adding to what's said above - thanks for the library, and it would be amazing if it were published to the PlatformIO registry.
